hash_practice/exercises.py

```diff
@@ -1,29 +1,131 @@
-
-def grouped_anagrams(strings):
-    """ This method will return an array of arrays.
+""" This method will return an array of arrays.
         Each subarray will have strings which are anagrams of each other
-        Time Complexity: ?
-        Space Complexity: ?
+        Time Complexity: O(n log n)

```

Comment:
Assuming that by `n` you mean the number of letters in a word, this is almost correct, but you have to make sure to account for the number of words in the list, which I'll call `m`.

In that case you would get O(m*n*log(n)) since there are `m` words in the list. However, since we're assuming that the words are English words, and thus they are limited in how long they can be (5 letters on average), `n` ends up being dwarfed in effect by `m`, making this O(m).